src/blockrelay/compactblock.cpp

```diff
@@ -86,8 +86,8 @@ void CompactBlock::FillShortTxIDSelector() const
     hasher.Write((unsigned char *)&(*stream.begin()), stream.end() - stream.begin());
     uint256 shorttxidhash;
     hasher.Finalize(shorttxidhash.begin());
-    shorttxidk0 = shorttxidhash.GetUint64(0);
-    shorttxidk1 = shorttxidhash.GetUint64(1);
+    shorttxidk0=shorttxidhash.GetUint64(0);
```

Comment:
You don't need to use `store` here like below?

Reply:
you are right will fix

Reply:
actually it turns out that the std::atomic is just for peer we are connecting to, `shorttxidk0` and `shorttxidk1` are our own salt and have not been turned into an atomic because they don't need locking, or at least hey are not currently annotated with GUARDED_BY.

those are defined here:
https://github.com/BitcoinUnlimited/BitcoinUnlimited/blob/release/src/blockrelay/compactblock.h#L191